packages/components/src/presentation-components/instance-filter-builder/InstanceFilterConverter.ts

```diff
@@ -65,8 +86,9 @@ function createExpressionFromGroup(group: GenericInstanceFilterRuleGroup): strin
   return `(${convertedConditions.join(` ${getGroupOperatorString(group.operator)} `)})`;
 }
 
-function createClassExpression(usedClasses: ClassInfo[]) {
-  return usedClasses.reduce((queryExpression, classInfo) => `${queryExpression}${queryExpression ? " OR " : ""}this.IsOfClass(${classInfo.id})`, "");
+/** @internal */
+export function createFilterClassExpression(usedClasses: ClassInfo[]) {
+  return `(${usedClasses.reduce((queryExpression, classInfo) => `${queryExpression}${queryExpression ? " OR " : ""}this.IsOfClass(${classInfo.id})`, "")})`;
```

Comment:
This should give the same result in an easier-to-read way
```suggestion
  return `(${usedClasses.map((classInfo) => `this.IsOfClass(${classInfo.id})`).join(" OR ")})`;
```